File: Data Structures & Algorithms/products-of-array-discluding-self/submission-2.java
class Solution {
    public int[] productExceptSelf(int[] nums) {
        int size = nums.length;
        int[] prodArr = new int[size];

        for(int i = 0; i < size; i++){
            int prod = 1;
            for(int j = 0; j < size; j++){
                if(i != j){
                    prod *= nums[j];
                }
            }
            prodArr[i] = prod;
        }
        return prodArr;
    }
}  

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-2.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        hashset = set(nums)
        longest = 0
        for num in nums:
            if (num-1) not in hashset:
                leng = 1
                while (num + leng) in hashset:
                    leng += 1
                longest = max(longest, leng)
        return longest

        


